syntax = "proto3";


package osrpc;

// The greeting service definition.
service OSRPC {
    // Sends a greeting
    rpc RPCInit (InitRequest) returns (InitReply) {
    }
    rpc RPCOpen (OpenRequest) returns (OpenReply) {
    }
}

// The request message containing the user's name.
message InitRequest {
}

// The response message containing the greetings
message InitReply {
    string rc = 1;
}


message OpenRequest {

}

message OpenReply {

}